README.md

```diff
@@ -67,36 +67,17 @@ Currently we are manually building an archive in xCode to distribute the app.
 4. Select Product > Archive to create the app archive.
 5. If your archive builds correctly it will walk you through pushing this to the app store from there. 
 
-### Running on device with local web
-1. Run web on your local machine as usual
-
-2. install ngrok
-```
-brew install ngrok
-```
-1. Create a free account here https://ngrok.com/ and obtain your auth token
-4. Configure ngrok with your token
-```
-ngrok config add-authtoken YOUR_TOKEN
-```
-3. run ngrok, note the public `Forwarding` domain listed
-```
-ngrok http 3000 # substitute port if necessary
-```
-4. Modify `.env` to point the WebView at the ngrok endpoint created above
-```
-SHAPESHIFT_URI=https://523d-135-134-162-133.ngrok.io # example ngrok domain
-```
-5. Run the app on your device from Xcode _or_ use the yarn script:
+### Running on Expo Go with local web
+- Run web, select the lan IP
+- Update your `EXPO_PUBLIC_LOCAL_URI` in the .env with the related IP
+- Run the mobile app using this command (note the --reset-cache option, it's necessary for Metro to reload the .env file):
 ```
-yarn ios:device
+npx expo start --reset-cache
 ```
-for now, the above runs `node_modules/.bin/react-native run-ios --device 'iPhone'` - substitute the name of your device for iPhone
-#### __Notes__
-- you may need to kill and reload the app after it's been completely loaded to get it to source content from metro
-- you may need to remove the line `messageManager.webviewRef?.reload()` in the useKeepAliveHook if the device has trouble downloading bundle.js before getting whacked.
-- you will need to manually add your ngrok domain to the `navigationFilter` to prevent opening in a browser. dev flag/mode?
-- as of 10/6: in your local web, move the entire contents of `react-app-rewired/headers/csps` out of their normal place and full restart web (yarn dev)
+- Install Expo Go
+- Launch ShapeShift by scanning the QR code
+- Select the `local` environment in the settings
```

Comment:
```suggestion
- Select the `localhost` environment in the settings
```

Reply:
Changed in [19d6b78](https://github.com/shapeshift/mobile-app/pull/109/commits/19d6b78d18a066a2297b173094cca097e6ae30b8)